src/marin/training/training.py

```diff
@@ -330,12 +330,16 @@ def _add_run_env_variables(env: dict):
     """
     env = deepcopy(env)
 
-    if "GIT_COMMIT" not in env:
+    git_commit = env.get("GIT_COMMIT") or os.environ.get("GIT_COMMIT")
+
+    if not git_commit:
         try:
-            env["GIT_COMMIT"] = levanter.infra.cli_helpers.get_git_commit()
+            git_commit = levanter.infra.cli_helpers.get_git_commit()
         except:  # noqa
             logger.warning("Could not infer git commit.")
 
+    env["GIT_COMMIT"] = git_commit or "unknown"
```

Comment:
i'm worried that "unknown" will make wandb sad, but i dunno

Reply:
Could be? I made it just empty again.